homeassistant_cli/plugins/system.py

```diff
@@ -19,3 +20,12 @@ def cli(ctx):
 def log(ctx):
     """Get errors from Home Assistant."""
     click.echo(api.get_raw_error_log(ctx))
+
+
+@cli.command()
+@pass_context
+def health(ctx: Configuration):
+    """Get system health from Home Assistant."""
+    frame = {'type': 'system_health/info'}
+
+    click.echo(api.wsapi(ctx, frame, False))
```

Comment:
good idea. should add a call to format here with some default columns so it works with the formatting flags.